docs/search-api.rst

```diff
@@ -0,0 +1,6 @@
+.. toctree::
+  :maxdepth: 1
+  :hidden:
+
+Search
+------
```

Comment:
Yeah but that's just a "trick" of CSS that we use for pages that actually exist.

It's moot since this isn't mean to be merged.